### ISSUE TITLE
Piwik Critical Issue RE: always_populate_raw_post_data=-1 in PHP.ini

### ISSUE THREAD

**Issue by fridgedoctor** [piwik/piwik#9741]
Under system check/required PHP, Im told to make the following change to php.ini: always_populate_raw_post_data=-1, but only if your version is PHP 5.6. Changing PHP to 5.4. would eliminate the need for the aforementioned change. I checked with my web server and I was using 5.6, so i switched it back to PHP 5.4 and removed the "always_populate_raw_post_data=-1 from the PHP.ini file, refresh piwik Dashboard, and now i have a 500 server error? Details say, "The website encountered an error while retrieving http://www.example.com/analytics/index.php?module=CoreHome&amp;action=index&amp;idSite=1&amp;period=week&amp;date=2016-02-08" One interesting not, If I am at the system check page and make these changes to PHP.ini, I will not get a server 500 error. The page will be nearly black except for the red PIWIK logo in upper left hand corner. So inorder for me to see anything in PIWIK version 2.15.0 im running, I need 5.6, which requires me to have "always_populate_raw_post_data=-1" in the php.ini file. The problem stems from the inability to view the map of my visitors

**Comment by tsteur** [comment]
Can you see some PHP errors in the logs when using PHP 5.4? It might not be related to `always_populate_raw_post_data `

**Comment by fridgedoctor** [comment]
Well, after looking at my server error logs, I see the following listed errors repeated several times each, from today's date back to the 20th of January, which is when i reinstalled piwik I believe.

/home3/example/public_html/analytics/error_log:
[06-Feb-2016 08:59:59 UTC] Error in Piwik (tracker): An unexpected website was found in the request: website id was set to '2' .

/home3/example/public_html/error_log:
[09-Feb-2016 21:32:02 UTC] PHP Warning:  Missing argument 1 for _e(), called in /home3/example/public_html/wp-content/themes/creativo/404.php on line 14 and defined in /home3/example/public_html/wp-includes/l10n.php on line 216.

There are some others listed, but they all pertain to my wordpress, within which I have the Piwik plugin installed, so it may be the cause of those errors.

When I look at my /analytics/error_log, I see the first one above repeated several times per day over several days as well as the following sort of interspersed:

[01-Feb-2016 18:59:44 UTC] PHP Parse error:  syntax error, unexpected '.', expecting '&' or variable (T_VARIABLE) in /home3/example/public_html/analytics/tmp/templates_c/d/f/dfe99bb3be8cd80e0a0d45b616c445f7ce75a1dfb0f85e21a7c319890d754e07.php on line 27

**Comment by tsteur** [comment]
> An unexpected website was found in the request: website id was set to '2' .

Did you maybe delete the website and something is still tracking data for this website?

> Missing argument 1 for _e(), called in /home3/example/public_html/wp-content/themes/creativo/404.php on line 14 and defined in /home3/example/public_html/wp-includes/l10n.php on line 216.

This seems to be not related to Piwik as you mentioned.

> /home3/example/public_html/analytics/tmp/templates_c/d/f/dfe99bb3be8cd80e0a0d45b616c445f7ce75a1dfb0f85e21a7c319890d754e07.php on line 27

Can you maybe delete all files within `templates_c`? eg `rm -r tmp/templates_c/*`

**Comment by fridgedoctor** [comment]
Did you maybe delete the website and something is still tracking data for this website?

If you mean my actual web site, no i did not delete it, nor did i delete it from my settings 
in piwik. I dont know what file the statement "An unexpected website was found in the request: website id was set to '2' " is referring to have even made that change.

Can you maybe delete all files within templates_c? eg rm -r tmp/templates_c/*

I renamed my templates_c folder, and also made a new empty templates_c folder.

I run Piwik, My visitor map keeps on loading and loading, some other widgets remain blank, and im still getting the same raw post data error under system check.

**Comment by tsteur** [comment]
Is it not possible for you to set `always_populate_raw_post_data=-1` with PHP 5.6?

**Comment by fridgedoctor** [comment]
<Sigh> Well, I added the "always_populate_etc." again I might add, and 
also while using PHP 5.6, and now when i run the check the critical 
error is gone, however the visitor map continues to load and load and no 
map ever appears. This is in Chrome, I have cleared the cache, not only 
restarted chrome but restarted pc, visitor map still loads and loads. 
But I see the problem is specific to chrome browser, because all 
widgets, including visitor map, load in IE, which I rarely use.

Is there some setting in Chrome im not aware of that I have set and 
should not, or vice versa?

Thank you for your help Tom
